src/fullnode.rs

```diff
@@ -176,12 +176,14 @@ impl Fullnode {
         }
 
         let bank = Arc::new(bank);
+        let blob_recycler = BlobRecycler::default();
         let mut thread_hdls = vec![];
 
         let rpu = Rpu::new(
             &bank,
             node.sockets.requests,
             node.sockets.respond,
+            &blob_recycler,
```

Comment:
This doesn't look right. Even though this is using Blobs, its content it totally different. I'd think this recycler can stay completely isolated.